src/pages/docs/infrastructure/workers/autoscaling-workers/index.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2024-08-22
+modDate: 2024-08-22
+title: Auto Scaling
+navOrder: 60
+---
+
+Workers are only utilized during the execution of a Deployment Process - which means they need to be able to handle high
```

Comment:
Again - this is task execution, right? Not a deployment process?

Reply:
Well, workers get used when deploying to targets which aren't literal computers - eg web services - so workers are part of the deployment process implicitly - or explicitly when dealing with steps like scripts.

Reply:
I'd agree with Liam here, specifying `Deployment Process` implies the exclusion of runbooks.

Reply:
updated in new PR